File: kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.js
/**
 * @param {number[]} candies
 * @param {number} extraCandies
 * @return {boolean[]}
 */
var kidsWithCandies = function(candies, extraCandies) {
  let most = 0;
  const result = [];
  for (let i = 0; i < candies.length; i++) {
    if (candies[i] > most) {
      most = candies[i];
    }
  }
  for (let i = 0; i < candies.length; i++) {
    if (candies[i] + extraCandies >= most) {
      result.push(true);
    } else {
      result.push(false);
    }
  }
  return result;
};

File: kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
      greatestCount = max(candies)
      res = []
      for i in range(len(candies)):
        if (candies[i] + extraCandies >= greatestCount):
          res.append(True)
        else:
          res.append(False)
      return res
        